share/backend_config.ini

```diff
@@ -32,12 +32,11 @@ database_file = /var/lib/zonemaster/db.sqlite
 #maximal_number_of_retries=3
 
 [LANGUAGE]
-locale = da_DK en_US fi_FI fr_FR nb_NO sv_SE
+locale = en_US da_DK fi_FI fr_FR nb_NO sv_SE
 
```

Comment:
I suggest a comment here that the first locale tag has a special status.

Reply:
Done.